src/cli/README_BR.md

```diff
@@ -353,10 +353,11 @@ Info per router:
   - Stub: Stub Router flag (indicates whether the router is a stub router)
 - Milliseconds since last received message from this router
 - Reachability flag: A router is marked as unreachable if it fails to respond to multiple Neighbor Solicitation probes.
+- Discover time: Duration interval since this router was first discovered. It is formatted as `{hh}:{mm}:{ss}` for hours, minutes, seconds, if the duration is less than 24 hours. If the duration is 24 hours or more, the format is `{dd}d.{hh}:{mm}:{ss}` for days, hours, minutes, seconds.
```

Comment:
nit: "Discover Time" doesn't seem quite right. My first thought when reading this was the time at which the router was discovered.

What about something like Router Age?

Reply:
Good idea. Changed it to use `mAge` and refer to it as "router age".